### ISSUE TITLE
src is not included in Dockerfile.test

### ISSUE THREAD

**Issue by antontsv** [vfarcic/books-ms#10]
In The DevOps 2.0 Toolkit book, `10.100.198.200:5000/books-ms-tests` container is built from `Dockerfile.test` [page 66]. 

Later on in the book this container is used to run `ServiceInteg.scala` using `sbt "testOnly *Integ"` [page 175].
`docker-compose -f docker-compose-dev.yml run --rm -e DOMAIN=http://10.100.198.201 integ` won't run ServiceInteg:

```
[info] Loading project definition from /home/vagrant/books-ms/project
[info] Set current project to books-ms (in build file:/home/vagrant/books-ms/)
[info] Passed: Total 0, Failed 0, Errors 0, Passed 0
[info] No tests to run for test:testOnly
[success] Total time: 1 s, completed Sep 15, 2016 3:24:01 AM
```

If src is included, for example `COPY src /source/src`, that SBT should execute the test:

```
[info] Loading project definition from /source/project
[info] Set current project to books-ms (in build file:/source/)
[info] Compiling 2 Scala sources to /source/target/scala-2.10/classes...
[info] 'compiler-interface' not yet compiled for Scala 2.10.6. Compiling...
[info]   Compilation completed in 35.563 s
[info] Compiling 2 Scala sources to /source/target/scala-2.10/test-classes...
[info] ServiceInteg
[info] 
[info] GET http://10.100.198.201/api/v1/books should
[info] + return OK
[info] 
[info] Total for specification ServiceInteg
[info] Finished in 19 ms
[info] 1 example, 0 failure, 0 error
[info] Passed: Total 1, Failed 0, Errors 0, Passed 1
[success] Total time: 75 s, completed Sep 16, 2016 6:32:16 AM
```

**Comment by vfarcic** [comment]
Hi,

By the look at the first output, you were probably not inside the directory
with the source code when you run the command.

Can you contact me on Skype (user: vfarcic) or HangOuts (user:
viktor@farcic.com)? I can walk you through it if you share your screen.

**Comment by antontsv** [comment]
Hi Viktor,

First of all thank you for an awesome book.

The steps were:
Build `cd` VM from https://github.com/vfarcic/ms-lifecycle/

```
vagrant ssh cd
// ... use docker-compose to start up the app on tcp://prod:2375
git clone https://github.com/vfarcic/books-ms.git
cd books-ms
docker-compose -f docker-compose-dev.yml run --rm -e DOMAIN=http://10.100.198.201 integ
```

Maybe `source` volume was not mounted as expected. 
I ran command out of cd:/home/vagrant/books-ms/; `src` directory is in `books-ms`.

As soon as included the src in the container itself, SBT invoked by run_tests.sh inside of the container ran fine. Plus prod VM had to be dumped up in RAM; `v.memory = 1024` was not enough to start JVM to compile and run ServiceInteg.

Since I rebuilt the container, there is nothing much to show; as it runs with new container.

Anyway I decided to open this issue just to let you know. The problem could be just because of different docker version that is out now and a year ago.
If no one else ran into issues, I think it is safe to close.
